ios/TeamHome/TeamHome/Models/Mock json/TeamExample.json

```diff
@@ -0,0 +1,10 @@
+ {
```

Comment:
Property names in JSON files should be double quoted. This isn't properly formed JSON.